jobs/Reservations/readme.md

```diff
@@ -0,0 +1,34 @@
+# Reservation Management
+
+**Fullstack App**
+
+> [!NOTE]
+> This task will involve a separate repository.
+
+Start by forking [the mews-reservations-int repo](https://github.com/MewsSystems/reservations-interview) and
+following the "Running the App" portion of the readme.
+
+```sh
+git clone git@github.com:MewsSystems/reservations-interview.git
+```
+
+You will be acting as if you are the lead engineer on a team, executing tasks from the `/backlog`. You are free to pick up
+any tasks you can reasonably do in ~90 minutes. You must indicate which tasks are complete and which are partially complete.
+
+Contribute as if you were intending to own and operate this product to serve customers on an ongoing basis on
+a **growing team**, in a fast paced scale up.
+
+Your submission must be in one Pull Request from a fork, and we encourage you to organize your commits so reviewers can
+step through commit by commit.
+
+<details>
+<summary>Technology</summary>
+
+The given technologies chosen for the starting point (.NET + React+TS) are what the majority of code at Mews is written in.
+
+You have **complete autonomy** for this product as a Lead Engineer and can implement any technology decisions you see fit for
+the tasks at hand. This includes adding / removing libraries, and adopting any design or architectural patterns.
+If you want, you could even _swap the stacks_ and start over with a different pair of stacks, or a full stack solution.
```

Comment:
Is this desirable? For those that haven't worked with c#/react/TS, is this just a pathway to 'do whatever you're comfortable with' and perhaps may not be testing them for compatibility with our own stack?

Reply:
I'm also thinking of our growing amount of stacks (RoR / future acquisitions), but perhaps let's remove this and give leeway to the tribe to restrict the stack? (ie if POS were to use this, they would want it in RoR I assume).

My brain, with this sentence, went to a typescript first fullstack solution that offers e2e type safety, or a C# full stack with blazor, both I think allow for us to evaluate the candidate and see what they could bring as a senior to a team - however... maybe less is more again. I can remove this suggestion. Perhaps an enterprising fellow can make a case to TA/Hiring manager for a different stack to offer that may align with our goals.

but yeah... if someone goes rust / go / htmx; that would not set them up for success at Mews imo.

Reply:
Done in ca10a09.